From 0ad7f0eeb0985df471d75d66500b7464b70b2d7a Mon Sep 17 00:00:00 2001
From: Vincent ABRIOU <vincent.abriou@st.com>
Date: Tue, 4 Feb 2020 17:01:01 +0100
Subject: [PATCH 1/1] TFLite: pip package: avoid bashism in build pip package
 script

Remove bashism in the script to avoid the script to crash when the build
is executed on a machine with a dash shell.

Signed-off-by: Vincent ABRIOU <vincent.abriou@st.com>
---
 tensorflow/lite/tools/pip_package/build_pip_package.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tensorflow/lite/tools/pip_package/build_pip_package.sh b/tensorflow/lite/tools/pip_package/build_pip_package.sh
index 1cb3866af7..8a3cb661ad 100755
--- a/tensorflow/lite/tools/pip_package/build_pip_package.sh
+++ b/tensorflow/lite/tools/pip_package/build_pip_package.sh
@@ -19,7 +19,7 @@ set -e
 PYTHON="${PYTHON:-python}"
 
 # Find where this script lives and then the Tensorflow root.
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 
 export TENSORFLOW_SRC_ROOT="${SCRIPT_DIR}/../../../.."
 export TENSORFLOW_VERSION=`grep "_VERSION = " "${TENSORFLOW_SRC_ROOT}/tensorflow/tools/pip_package/setup.py" | cut -d'=' -f 2 | sed "s/[ '-]//g"`;
@@ -41,9 +41,9 @@ cp "${TFLITE_ROOT}/tools/pip_package/MANIFEST.in" "${BUILD_ROOT}"
 # Build wheel file.
 cd "${BUILD_ROOT}"
 
-if [[ "${TENSORFLOW_TARGET}" == "rpi" ]]; then
+if [ "${TENSORFLOW_TARGET}" = "rpi" ]; then
   ${PYTHON} setup.py bdist_wheel --plat-name=linux-armv7l
-elif [[ "${TENSORFLOW_TARGET}" == "aarch64" ]]; then
+elif [ "${TENSORFLOW_TARGET}" = "aarch64" ]; then
   ${PYTHON} setup.py bdist_wheel --plat-name=linux-aarch64
 else
   ${PYTHON} setup.py bdist_wheel
-- 
2.17.1

